ethers/events.nim

```diff
@@ -33,14 +34,26 @@ func decode*[E: Event](decoder: var AbiDecoder, _: type E): ?!E =
   decoder.finishTuple()
   success event
 
+func isSupported(T: type): bool =
```

Comment:
Eventually I would like to see this based on solidityType(T), so that a custom type that is e.g. represented as a uint8 in solidity is also supported.

Reply:
`solidityType(T)` returns a `string`. How do you envision it being used here?

Reply:
I pushed a commit to show you what I mean. Feel free to remove it if you don't like it.

Reply:
Makes complete sense. Converting the types to strings was what I had in mind but wasn't sure if that's what you were looking for. Nice job!